### PR TITLE
fix dependency

### PR DESCRIPTION
**Pull request by rdixonbhw**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -31,5 +31,5 @@ dependencies {
     compile 'com.facebook.react:react-native:0.20.+'
     compile fileTree(include: ['*.jar'], dir: 'libs')
     testCompile 'junit:junit:4.12'
-    compile 'com.github.afollestad.material-dialogs:commons:0.8.6.2'
+    compile 'com.afollestad.material-dialogs:commons:0.8.6.2
```
Comment (padupuy) [inline android/build.gradle]: You forgot a quote `'` at the end of the line L.34

Comment (rdixonbhw) [inline android/build.gradle]: Whoops, my bad. Guess that's what happens when I build out of one repo and make PRs out of another.

**Comment by rdixonbhw** [comment]
So while I'm using an identical build.grade file for this as part of a React Native project, building this alone gives `Error:Module 'com.afollestad.material-dialogs:commons:0.8.6.2' depends on one or more Android Libraries but is a jar`, despite it working when part of another project. Not sure where to go from here.

**Comment by SkyzohKey** [comment]
Error on first compile, patched with this PR, now works. This must be merged :)

**Comment by vonovak** [comment]
thanks @rdixonbhw !

**Pull request merged**